transport/src/main/java/io/scalecube/transport/TransportConfig.java

```diff
@@ -52,13 +60,14 @@ public String toString() {
     private int port = DEFAULT_PORT;
     private boolean useNetworkEmulator = DEFAULT_USE_NETWORK_EMULATOR;
     private int connectTimeout = DEFAULT_CONNECT_TIMEOUT;
+    private MessageCodec messageCodec;
```

Comment:
i guess this could be changed to:
private MessageCodec messageCodec = DEFAULT_MESSAGE_CODEC